source/includes/_release-notes.html.md

```diff
@@ -0,0 +1,77 @@
+# Product Release Notes
+
+These release notes combine releases for the web (BRAIN Dashboard), the backend services (Bonsai AI Engine), and Bonsai's API. All releases are tagged by date for the web, backend, and API. Releases are based on version for the SDKs and CLI which are available on GitHub.
+
+Release Notes for CLI: [https://github.com/BonsaiAI/bonsai-cli/blob/master/CHANGELOG.md]()
+
+Release Notes for Python SDK (bonsai-ai): [https://github.com/BonsaiAI/bonsai-sdk/blob/master/bonsai-ai/CHANGELOG.md]()
+
```

Comment:
As mentioned, there was a `2018.03.07` release, which notably had a number of bug fixes for resume training.